rules/network/command_and_control_cobalt_strike_beacon.toml

```diff
@@ -40,7 +40,9 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset: (network_traffic.tls or network_traffic.http) AND destination.domain:/[a-z]{3}.stage.[0-9]{8}\..*/
+((event.category: (network or network_traffic) and type: (tls or http) and network.transport: tcp)
```

Comment:
do packetbeat and NPC not populate `type` and `network.transport` the same way?

Reply:
If I understand correctly, `type` could be anything between transport and application layer, whereas `network.transport` is mainly transport layer so we can only specify UDP or TCP. Ideally, if `type` is in both schemas and we define `tls` then by default it would `tcp` and we can drop the `network.transport` filter. I will double check this.